core/templates/domain/utilities/autogenerated-audio-language.model.ts

```diff
@@ -43,13 +40,8 @@ export class AutogeneratedAudioLanguage {
     this.speechSynthesisCode = speechSynthesisCode;
     this.speechSynthesisCodeMobile = speechSynthesisCodeMobile;
   }
-}
 
-@Injectable({
-  providedIn: 'root'
-})
-export class AutogeneratedAudioLanguageObjectFactory {
-  createFromDict(
+  static createFromDict(
```

Comment:
Ditto, can you please share the link to the file where this is used? (I wanted to check whether we can use the constructor instead of passing a dict*)

Reply:
[language-util.service.ts Line 87](https://github.com/oppia/oppia/pull/10713/files#diff-569ba9a2f5e25eee34f133695b0d8149)

Reply:
Thanks for sharing the link @darkpsychic, currently I'm unable to think about a better structure for this. For now, let's leave it as is and if required we will change it in the future (Considering this is just an issue with the audio language services.)